### PR TITLE
docs(CONTRIBUTING.md): change type descriptions for less confusion

### PR DESCRIPTION
**Pull request by ghost**
Fixed commit message of #3463

### CODE REVIEW

**Comment by tux3** [comment]
Your last commit seems to break the naming conventions again!
Note that you can edit the last commit with "git commit --amend" instead of creating a new one (only if nobody has pulled your branch yet!).

**Comment by ghost** [comment]
I'm so dumb...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -84,8 +84,9 @@ Must be one of the following:
 * **fix**: A bug fix
 * **docs**: Documentation only changes
 * **style**: Changes that do not affect the meaning of the code (white-space,
-  formatting, etc)
-* **refactor**: A code change that neither fixes a bug nor adds a feature
+  formatting, etc), but change the style to a more appropriate one
+* **refactor**: A code change that only improves code readability 
+  and reduced complexity, without changing any functionality
```
Comment (zetok) [inline CONTRIBUTING.md:89]: `reduced` ?

**Comment by tux3** [comment]
Okay I'll try to save you some fighting with Git and just apply a typo fix on top of your PR :)

**Comment by ghost** [comment]
I thought, that I fixed it earlier, it seems I've failed to do it properly...again...

**Pull request merged**